### PR TITLE
multiwan: added object references to rule creation/edit

### PR DESCRIPTION
**Pull request by Tbaile**
Ref:
- https://github.com/NethServer/nethsecurity/issues/671

### CODE REVIEW

File: multiwan.rst [hunk 1]
```diff
@@ -56,6 +56,11 @@ Here's how you can create a custom rule:
 
 4. Specify traffic type: define the criteria for the traffic you want to customize. This can include the source IP address, destination IP address,
    specific protocols, ports, or any combination of these factors. By specifying these parameters, you narrow down the scope of the rule to a specific type of traffic.
+   With the Source Address and Destination Address fields, you can pick from the following options:
+
+   - Enter an address or range: specify a single IP address or a CIDR. Only IPv4 is supported.
+   - Any address: select this option to include all addresses.
+   - Select a firewall object: choose from the list of predefined firewall objects.
```
Comment (Tbaile) [inline multiwan.rst:63]: Shall we add all the list of the possible entries available?

File: multiwan.rst [hunk 2]
```diff
@@ -56,6 +56,11 @@ Here's how you can create a custom rule:
 
 4. Specify traffic type: define the criteria for the traffic you want to customize. This can include the source IP address, destination IP address,
    specific protocols, ports, or any combination of these factors. By specifying these parameters, you narrow down the scope of the rule to a specific type of traffic.
+   With the Source Address and Destination Address fields, you can pick from the following options:
+
+   - Enter an address or range: specify a single IP address or a CIDR. Only IPv4 is supported.
+   - Any address: select this option to include all addresses.
```
Comment (gsanchietti) [inline multiwan.rst:62]:
```suggestion
   - Any address: select this option to match any address.
```

File: multiwan.rst (hunk 1 above)
Comment (gsanchietti) [reply multiwan.rst:63]: Not needed

**Review by andre8244 — APPROVED** [review]

**Review by gsanchietti — APPROVED** [review]
Do not merge until the release of the package

**Pull request merged**